src/webview/flink-statement-results.html

```diff
@@ -89,6 +89,8 @@ <h4>Statement details</h4>
               placeholder="Search across results…"
               type="search"
               size="50"
+              data-prop-value="this.search()"
+              data-on-change="this.search(event.target.value)"
```

Comment:
Turns out -- this is needed to preserve the search input text when you tab out and tab back in.

Reply:
Works good now, thanks!